### PR TITLE
chore: update secrets in helm

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

File: helm/templates/configmap.yaml [hunk 1]
```diff
@@ -40,13 +41,4 @@ data:
   S3_ENDPOINT: {{ .Values.env.S3.endpoint | quote }}
   S3_BUCKET: {{ .Values.env.S3.bucket | quote }}
   S3_FORCE_PATH_STYLE: {{ .Values.env.S3.forcePathStyle | quote }}
-  S3_SSL_ENABLED: {{ .Values.env.S3.sslEnabled | quote}}
```
Comment (cl-shaharshavit-rnd) [inline helm/templates/configmap.yaml:43]: why s3 ssl was removed???

File: helm/templates/gw-secret.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.env.gateway.auth.enabled -}}
```
Comment (cl-shaharshavit-rnd) [inline helm/templates/gw-secret.yaml:1]: add support for signature without auth

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]

File: helm/templates/configmap.yaml (hunk 1 above)
Comment (AnguIar) [reply helm/templates/configmap.yaml:43]: oops

File: helm/templates/gw-secret.yaml (hunk 2 above)
Comment (AnguIar) [reply helm/templates/gw-secret.yaml:1]: auth is always needed
signature is needed in some cases

**Review by cl-shaharshavit-rnd — APPROVED** [review]

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

**Pull request merged**